simon_says/ui/src/registerServiceWorker.js

```diff
@@ -0,0 +1,117 @@
+// In production, we register a service worker to serve assets from local cache.
```

Comment:
This file needs proper acknowledgement. Who did the UI?

Reply:
Deep

Reply:
according to Deep this can be removed.